modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -7,28 +7,25 @@
 [id="oc-mirror-creating-image-set-config_{context}"]
 = Creating the image set configuration
 
-Before you can use the oc-mirror plugin to mirror image sets, you must create an image set configuration file. This image set configuration file defines which {product-title} releases, Operators, and other images to mirror, along with other configuration settings for the oc-mirror plugin.
+To use the oc-mirror plugin to mirror image sets, you must create an `ImageSetConfiguration` file. This file specifies the following information:
 
-You must specify a storage backend in the image set configuration file. This storage backend can be a local directory or a registry that supports link:https://docs.docker.com/registry/spec/manifest-v2-2[Docker v2-2]. The oc-mirror plugin stores metadata in this storage backend during image set creation.
+* The {product-title} releases, Operators, and additional images for mirroring
 
-[IMPORTANT]
-====
-Do not delete or modify the metadata that is generated by the oc-mirror plugin. You must use the same storage backend every time you run the oc-mirror plugin for the same mirror registry.
-====
+* Various configuration settings for the oc-mirror plugin v1
 
 .Prerequisites
 
 * You have created a container image registry credentials file. For instructions, see "Configuring credentials that allow images to be mirrored".
 
 .Procedure
 
-. Use the `oc mirror init` command to create a template for the image set configuration and save it to a file called `imageset-config.yaml`:
+. To create a template for the image set configuration, Uuse the `oc mirror init` command and save the informationit to a file called `imageset-config.yaml`. 
 +
 [source,terminal]
 ----
 $ oc mirror init <--registry <storage_backend> > imageset-config.yaml <1>
```

Comment:
can you help check if this is rendering properly ?

Reply:
this part- $ oc mirror init <--registry <storage_backend> > imageset-config.yaml <1>?

Reply:
actually this is confusing <---registry < storage-backend> > imageset-config.yaml, do not think all this is required.. Simply having oc-mirror --init should be sufficient is what i believe

Including sample output for your refernce:
```
[fedora@preserve-fedora-yinzhou knarra]$ ./oc-mirror init
kind: ImageSetConfiguration
apiVersion: mirror.openshift.io/v1alpha2
storageConfig:
  local:
    path: ./
mirror:
  platform:
    channels:
    - name: stable-4.16
      type: ocp
  operators:
  - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.16
    packages:
    - name: serverless-operator
      channels:
      - name: stable
  additionalImages:
  - name: registry.redhat.io/ubi8/ubi:latest
  helm: {}
```